src/modules/auth/api/auth.mutations.controllers.ts

```diff
@@ -1,4 +1,27 @@
-import { Controller } from "@nestjs/common";
+import { Body, Controller, Inject, Post, UsePipes, ValidationPipe } from "@nestjs/common";
+import { Paths } from "src/core/api/routes";
+import { LoginInput } from "src/modules/auth/api/auth.mutations.inputs";
+import { ApiAuth } from "src/modules/auth/api/auth.types";
+import { AuthServiceDefinition, AuthServiceName } from "src/modules/auth/domain/auth.service.types";
 
-@Controller("auth-mutations")
-export class AuthMutationsController {}
+@Controller(Paths.auth.root)
+@UsePipes(new ValidationPipe())
+export class AuthMutationsController {
+	constructor(@Inject(AuthServiceName) private authService: AuthServiceDefinition) {}
+
+	@Post(Paths.auth.mutations.login)
+	async login(@Body() input: LoginInput): Promise<ApiAuth> {
+		const serviceUser = await this.authService.login({
+			username: input.username,
+			password: input.password,
+		});
+
+		const apiUser: ApiAuth = {
```

Comment:
It doesn't make sense to call a constant apiUser if it's type is ApiAuth. Neither of these words match with each other in any context.

Reply:
A good name for this type would be `ApiAuthUser`